src/deva/cli/__init__.py

```diff
@@ -141,8 +140,29 @@ def deva(
             config.data.clear()
         else:
             ctx.fail(f"Error loading config: {config.path}\n{e}")
-
     app = Application(terminator=ctx.exit, config_file=config, enable_color=color, interactive=interactive)
+
+    # Telemetry config
+    if not interactive:
+        if app.config_file.data.get("telemetry").get("user_consent") is None:
```

Comment:
I think since both the consent and API key are not technically configuration we should simply treat this as a managed JSON file in the cache directory instead. What do you think?